ikkinz/player.lua

```diff
@@ -12,6 +12,10 @@ local instance_ver = eq.get_zone_instance_version();
 			e.self:Message(15, "The markings on the Overseer's Geostone line up with the markings on the door. It rumbles to life as it opens for you.");			
 		elseif (e.self:HasItem(60225)==true and instance_ver==4) then
 			e.self:Message(15, "The glyphs on Axtekilitek and the door glow momentarily before the doors slide open.");			
+		elseif instance_ver == 2 then -- group trial 3
```

Comment:
I'd like us to use constants for these, we have constants available in `./lua_modules/constants/instance_versions.lua`

```lua
instance_version = {
    the_nektulos_forest = 1,
    the_bazaar = 1,
    stillmoon_temple_sickness_of_the_spirit = 1,
    stillmoon_temple_animated_statue_plans = 2,
    stillmoon_temple_best_laid_plans = 3,
    stillmoon_temple_diseased_pumas = 4,
    stillmoon_temple_guardian_of_the_sands = 5,
    stillmoon_temple_scales_of_justice = 6,
...
```

Reply:
There's also a generator SQL you can find in the constants file itself if you have versions not available

Reply:
The database doesn't have the ikkinz zone versions so I added the constants manually